src/main/resources/static/styles.css

```diff
@@ -32,6 +32,6 @@
 
 #map {
     height: 80vh;
-    width: 80vw;
+    width: 75vw;
```

Comment:
why did this width change?

Reply:
Ah, I was messing with placement of the directions, and did not mean to leave it at 75. Fixed!